extensions/binding/org.eclipse.smarthome.binding.onewire.test/src/test/java/org/eclipse/smarthome/binding/onewire/internal/MultisensorThingHandlerTest.java

```diff
@@ -55,6 +55,7 @@ public void setup() {
         thingConfiguration.put(CONFIG_ID, TEST_ID);
         thingProperties.put(PROPERTY_SENSORCOUNT, "1");
         thingProperties.put(PROPERTY_THING_TYPE_VERSION, "1");
+        thingProperties.put(PROPERTY_MODELID, "AMS");
```

Comment:
This will fix the test, but the NPE will still occur in normal operation (see below).